tests/test_preannotated.nf.test.snap

```diff
@@ -103,14 +75,26 @@
         },
         "timestamp": "2024-07-04T22:23:20.593242"
     },
+    "argnorm_deeparg": {
+        "content": [
+            "sample_1.normalized.tsv:md5,4a1807992bab9ed4ee9384d237d77e59",
+            "sample_2.normalized.tsv:md5,a922864845c3a894fa8f5c43cf2e23dc",
+            "sample_3.normalized.tsv:md5,1bb503373085539ae81c31edfdb2ef34"
+        ],
+        "meta": {
+            "nf-test": "0.8.4",
+            "nextflow": "24.04.3"
+        },
+        "timestamp": "2024-07-18T12:32:12.966214504"
+    },
     "macrel": {
         "content": [
             "sample_1.macrel.smorfs.faa.gz:md5,9cddad1e4b6dbcb76888f1a87db388ec",
             "sample_2.macrel.smorfs.faa.gz:md5,e055dd2a9e44f3dcaa8af7198600349c",
             "sample_3.macrel.smorfs.faa.gz:md5,9cddad1e4b6dbcb76888f1a87db388ec",
-            "sample_1.macrel.all_orfs.faa.gz:md5,c276fb1ec494ff53ded1e6fc118e25b9",
-            "sample_2.macrel.all_orfs.faa.gz:md5,e75e434a30922d80169d0666fd07e446",
-            "sample_3.macrel.all_orfs.faa.gz:md5,c276fb1ec494ff53ded1e6fc118e25b9",
+            "sample_1.macrel.all_orfs.faa.gz:md5,382803053f9532d3c970c5db940eb616",
```

Comment:
Why did those change for macrel? Maybe thats why the ci tests are failing because they changed here.